### PR TITLE
Add Amazon RWS Record

### PR DESCRIPTION
**Pull request by rareog007**

### CODE REVIEW

**Comment by ratchanonsuttawas** [comment]
7ff2ca02253d27ac6227458de8d1cddb1174b0e8

File: related_website_sets.JSON
Comment (ratchanonsuttawas) [inline related_website_sets.JSON:1]: Summit feedback and approve merging these changes

[the remark this answers is not in the document]
Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > related_website_sets.JSON

Comment (ratchanonsuttawas) [reply related_website_sets.JSON:1]: > Summit feedback and approve merging these changes

File: related_website_sets.JSON [hunk 1]
```diff
@@ -952,6 +952,51 @@
      "rationaleBySite": {
         "https://drimer.travel": "The travel portal for Drimer"
      }
-  }
```
[the remark this answers is not in the document]
Comment (ratchanonsuttawas) [reply related_website_sets.JSON:955]: > 26188d7676ac3f1e6eba4c62ad795526e5649496

**Comment by Sufailmc** [comment]
> [7ff2ca0](https://github.com/GoogleChrome/related-website-sets/commit/7ff2ca02253d27ac6227458de8d1cddb1174b0e8)

**Comment by cfredric** [comment]
This appears to be a duplicate of https://github.com/GoogleChrome/related-website-sets/pull/539. Closing in favor of that one.

**Pull request closed without merging**